packages/ocaml-option-fp/ocaml-option-fp.1/opam

```diff
@@ -8,9 +8,11 @@ homepage: "https://opam.ocaml.org"
 bug-reports: "https://github.com/ocaml/opam-repository/issues"
 license: "CC0-1.0+"
 depends: [
-  "ocaml-variants" {post & (os = "linux" & >= "4.12.0~" | os = "macos" & >= "5.3.0~")}
+  "ocaml-variants" {post & ((os = "linux" & arch = "x86_64" & >= "4.12.0~") |
+                            (os = "macos" & arch = "x86_64" & >= "5.3.0~") |
+                            (arch = "arm64" & >= "5.4.0~"))}
```

Comment:
This would include an eventual windows or bsd arm64. The unspecified os seems to be the only difference between this PR and #27881. Which one is the correct one?

Reply:
it would not since the `available` field does forbid any other OSs